### ISSUE TITLE
jsSourceDirectories is broken on sbt 1.3.0

### ISSUE THREAD

**Issue by mushtaq** [scalacenter/scalajs-bundler#315]
I have this setting in my current project: 
`Compile / jsSourceDirectories += baseDirectory.value / "html"` 

It works well with sbt-1.2.8, but with sbt-1.3.0 fails with:

```
(Compile / npmInstallJSResources) java.lang.IllegalArgumentException: requirement failed: Source file '/Users/mushtaq/projects/tmt/msocket/example-service/example-service-app-js/html' is a directory.
```

**Comment by saksmt** [comment]
My guess it's because Path.allSubpaths returns not only files, but directories too in scalajsbundler.sbtplugin.NpmUpdateTasks.scala:80. And output should be filtered: `.filter(_._1.isDirectory)`. IMHO it would be better to just write that code by hand

**Comment by saksmt** [comment]
Hm, looks like it's already fixed in master and we only need published artifact for this. Any maintainers still alive?

**Comment by julienrf** [comment]
The fix is available in version 0.16.0.
